octobot_trading/personal_data/orders/decimal_order_adapter.py

```diff
@@ -188,7 +188,11 @@ def decimal_check_and_adapt_order_details_if_necessary(quantity, price, symbol_m
 
             # check total_order_price not < min_cost
             if not personal_data.check_cost(float(total_order_price), min_cost):
-                return []
+                symbol = f"{symbol_market['symbol']} - " if 'symbol' in symbol_market else ""
+                raise errors.MinimumOrderCostNotReachedError(
```

Comment:
Then it should be catch if necessary everywhere this method is called

Reply:
I added exceptions, also in the PR in octobot tentacles ->  [Order] handle MinimumOrderCostNotReachedError